ntbs-service/wwwroot/css/search.scss

```diff
@@ -23,3 +23,9 @@
     margin-left: auto;
     margin-bottom: 0;
 }
+
+.search-page {
+    .validation-summary-errors {
+        display: none;

```

Comment:
this is so that the error summary is not shown when the page first loads (and the search doesn't have any of the required parameters) , the errors are then correctly mapped when a search is done

Reply:
Hmm, seems odd to do this in the css... Seems like we want to be able to differentiate between the states in view logic?